python/lsst/pipe/analysis/colorAnalysis.py

```diff
@@ -634,6 +642,21 @@ def transformCatalogs(self, catalogs, transforms, flagsCats=None, hscRun=None):
 
         return new
 
+    def plotGalacticExtinction(self, byFilterCats, filenamer, dataId, butler=None,
+                               camera=None, tractInfo=None, patchList=None, hscRun=None):
+        for ff in byFilterCats:
+            qMin = np.nanmean(byFilterCats[ff]["A_" + ff]) - 6.0*np.nanstd(byFilterCats[ff]["A_" + ff])
+            qMax = np.nanmean(byFilterCats[ff]["A_" + ff]) + 6.0*np.nanstd(byFilterCats[ff]["A_" + ff])
+            shortName = "galacticExtinction_" + ff
+            self.log.info("shortName = {:s}".format(shortName))
+            self.AnalysisClass(byFilterCats[ff], byFilterCats[ff]["A_" + ff],
```

Comment:
ff is not particular descriptive, might be easier to understand if it was something else.